modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,541 @@
++
+[NOTE]
+====
+* Once `enableCrossClusterVersioning` is enabled, it cannot be disabled.
+As a result, HLV also cannot be disabled.
+
+* When you add a new cluster to the existing replication topology, set `enableCrossClusterVersioning` to `true` in each bucket of the new cluster, to activate conflict logging. Because only the documents mutated after enabling the `enableCrossClusterVersioning` property on both the source and target clusters, are considered for conflict logging.
+====
+
+. When creating buckets and collections, designate at least one bucket as a conflict bucket and one collection in it as a xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-collection[conflict collection], on every cluster, for logging conflicts.
+Use the following:
+
+** xref:manage:manage-xdcr/create-xdcr-replication.adoc#xdcr-ui-settings-for-conflict-logging
+[Replication Settings for XDCR Conflict Logging], to set from UI.
+
+** xref:rest-api:rest-xdcr-create-replication.adoc[Creating a Replication] by xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#configure-conflictlogging-settings[Configuring the conflictLogging Settings], to set through REST API.
+
+. xref:learn:clusters-and-availability/xdcr-viewing-conflict-logs.adoc[Viewing Conflict Logs]: When XDCR detects conflicts, it logs xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record[Conflict Record] into the conflict collection specified by you.
+You can access and review the logged conflicts.
+
+[#conflict-collection]
+== Conflict Collection
+
+A conflict collection is a regular collection that you create to store the xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record[Conflict Record] (Conflict Record Document, and a copy of the source and target documents responsible for the conflict) from the affected clusters of the XDCR topology.
+
+For information about configuring a collection to log conflicts, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#configure-conflictlogging-settings[Configuring the conflictLogging Settings].
+
+* As conflict collection is a regular collection, you can use regular RBAC to control standard access and programmatic access to the documents in the collection.
+
+* You can store conflict collection in the source bucket or in a separate collection in a bucket that's not used in any XDCR replications.
+
+* XDCR can write into conflict collections associated with the source cluster, however, XDCR cannot modify the conflict collections in any target clusters.
+
+NOTE: XDCR adds a hidden system xattribute flag called `_xdcr_conflict` into each of the three documents in a xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record[Conflict Record] stored in conflict collection, which prevents these documents (conflict logs) from replicating to the target.
```

Comment:
NOTE: XDCR adds a system xattribute called `_xdcr_conflict` into each of the three documents in the xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-record[Conflict Record], which prevents these documents (conflict logs) from being replicated.